### DISCUSSION TITLE
I want to sort the search results by the file's editing time or creation time. What should I do?

### DISCUSSION THREAD

**Discussion by tyf2018** [tadashi-aikawa/obsidian-another-quick-switcher#163] [Q&A] [answered]
The Grep function is currently working properly and also quite fast. Now there is a requirement: I want to sort the search results by the file's editing time or creation time. What should I do?

Best regards.

**Comment by tadashi-aikawa** [comment]
@tyf2018 
Sorry for the late answer 🙇 

> search results by the file's editing time or creation time. What should I do?

You can sort them by specifying [Sort priorities](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher#sort-priorities). 

![image](https://user-images.githubusercontent.com/9500018/236669005-fe550284-636b-4c85-92b6-5a2a3cf7bd70.png)

**Comment by tyf2018** [comment]
Thanks，but My point is that I don't seem to see any setting options in Grep search.

**Reply by tadashi-aikawa** [reply]
Oh.. I misread. We can't do it on the latest version; however, I want to implement that feature 👍

**Reply by tadashi-aikawa** [reply, marked as answer]
Continue 👉 https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/169
